smfgen

```diff
@@ -21,50 +25,42 @@ function checkAndApplyContext(prefix, conf, applyto)
 	var s = (prefix ? prefix + '.' : '');
 
 	if (conf.hasOwnProperty('user')) {
-		mod_assert.equal(typeof (conf.user), 'string',
-		    '"' + s + 'user" property must be a string');
+		assert.string(conf.user, s + 'user');
```

Comment:
Don't think I've ever seen old-style assert code before. Wow is assert-plus nicer!

Reply:
oh god ha, yeah. it was fun.